bin/compress_tif_jp2.pl

```diff
@@ -43,17 +43,13 @@
 
     # try to compress the TIFF -> JPEG2000
     print("Compressing $infile to $outfile\n");
-    my $kdu_compress = get_config('kdu_compress');
-    die("You must correctly configure the path to kdu_compress in the feed configuration\n") 
-    if not defined $kdu_compress or !-x $kdu_compress;
+    my $grk_compress = get_config('grk_compress');
+    die("You must correctly configure the path to grk_compress in the feed configuration\n") 
+    if not defined $grk_compress or !-x $grk_compress;
 
-    # Settings for kdu_compress recommended from Roger Espinosa. "-slope"
-    # is a VBR compression mode; the value of 42988 corresponds to pre-6.4
-    # slope of 51180, the current (as of 5/6/2011) recommended setting for
-    # Google digifeeds.
-    system(qq($kdu_compress -quiet -i '$infile' -o '$outfile' Clevels=$levels Clayers=8 Corder=RLCP Cuse_sop=yes Cuse_eph=yes "Cmodes=RESET|RESTART|CAUSAL|ERTERM|SEGMARK" -no_weights -slope 42988))
+    system(qq($grk_compress -i "$infile" -o "$outfile" -p RLCP -n $levels -SOP -EPH -M 62 -I))
```

Comment:
So in the end we don't need a slope equivalent for grk_compress? (the slack thread ends on an open question...)

Reply:
We can provide a target PSNR or compression rate; I'm looking to see what typical PSNR we end up with for scanned book pages with the kakadu slope rate.